services/meta/association.md

```diff
@@ -3,29 +3,40 @@ This service allows creating a direct association between a source device (e.g.
-
 #### Service Name
 
 `association`
 
 #### Interfaces
 
-Type | Interface                  | Value type | Description
------|-----------------------     |------------|------------
-in   | cmd.association.add        | object     | Add members to the group
-in   | cmd.association.delete     | object     | Remove association for a single member
-in   | cmd.association.delete_all | string     | Remove associations for all group members
-in   | cmd.association.get_report | string     | Get all members of a group
-out  | evt.association.report     | object     | Response for `get_report`
-
-#### Service Props
-
-Name             | Supported Values           | Description
------------------|----------------------------|-------------
-`in_services`   | `["out_bin_switch", "out_lvl_switch", "color"]` | List of services that can be controlled, e.g. on a lamp
-`out_services`   | `["out_bin_switch", "out_lvl_switch", "color"]` | List of services the device can control, e.g. on a button
-
+Type | Interface                  | Value type | Properties     | Description                                                                                                                    |
+-----|----------------------------|------------|----------------|--------------------------------------------------------------------------------------------------------------------------------|
+in   | cmd.association.add        | object     |                | Add members to a specific association group.                                                                                   |
+in   | cmd.association.delete     | object     |                | Remove a single memeber from an association group.                                                                             |
+in   | cmd.association.delete_all | int        |                | Remove all memebers from an association group.                                                                                 |
+in   | cmd.association.get_report | int        |                | Request report of a specific association group.                                                                                |
+out  | evt.association.report     | object     | `max_supports` | Reports members of a specific association. Property `max_supports` defines how many devices can be added to this group.        |
+in   | cmd.groups_info.get_report | null       |                | Z-Wave only. Request detailed information about supported association groups.                                                  |
+out  | evt.groups_info.report     | object     |                | Z-Wave only. Returns a list of supported associations groups with descriptions. See [`association_info_object`](#definitions). |
+
+### Definitions
+
+* `association_info_object` is an object describing a specific association group
```

Comment:
`association_info_object` is an documentation only name, no need to nitpick it, but _object suffix is mostly useless here as you have object. Also a colon at the end of sentence?